utils/versions.py

```diff
@@ -1,5 +1,5 @@
 import modulefinder
-import git
+#import git
```

Comment:
must be kept

Reply:
need to clarify

Reply:
OK it was a temporary issue from SW long time ago, need to get it back

Reply:
I think you can just find the commit(s) where these changes have been introduced in this PR and just revert them by pushing to this branch.